contracts/smart-wallet/non-upgradeable/Account.sol

```diff
@@ -167,6 +172,24 @@ contract Account is
         return 0;
     }
 
+    /// @notice Registers a signer in the factory.
+    function _setupRole(bytes32 role, address account) internal virtual override {
+        super._setupRole(role, account);
```

Comment:
What if we setup signer role to the admin on initialize? That way getSignersOfAccount can return the owner too